TabloidMVC/Repositories/UserProfileRepository.cs

```diff
@@ -53,5 +54,36 @@ public UserProfile GetByEmail(string email)
                 }
             }
         }
+
+        public UserProfile Add(UserProfile user)
+        {
+            using (var conn = Connection)
+            {
+                conn.Open();
+                using (var cmd = conn.CreateCommand())
+                {
+                    cmd.CommandText = @"
+                        INSERT INTO Userprofile (
+                            FirstName, LastName, DisplayName, Email, CreateDateTime,
```

Comment:
Has the default value for IsDeleted been established with the database?

Reply:
yes